README.md

```diff
@@ -58,3 +58,20 @@ end
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and the created tag, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+
+## VCR Cassettes
+
+VCR gem is used to record the results of the API calls for the tests.  If you need to
```

Comment:
VCR allows you to obscure automatically through configuration. (See sul_pub for example.)

Reply:
Yeah, I know - some of the auto obfuscation requires you to specify the string to be obscured, which in sul-pub comes from Settings (which we don't have in the gem).  But I'll check if there is a more clever way to do this, like regex.